### PR TITLE
PropType

### PR DESCRIPTION
**Pull request by xcatlkd**
Added a type check to Title.js.
Checks if the type of the props.children attribute is a number.
Obviously super useful, but does throw an error in the console if another type is passed.

### CODE REVIEW

**Comment by wbobeirne** [comment]
Nice work. But maybe you could make it useful by using it on your Font class? Size should always be a number, but color should always be a string.

**Review by wbobeirne — CHANGES_REQUESTED** [review]
Try to put the prop types on Font for the right requirements

**Comment by xcatlkd** [comment]
That is a more reasonable use case.

**Comment by xcatlkd** [comment]
Code updated. Changes made to Font.js to check PropTypes on both color (string) and size (number).

File: js/components/Font.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		return (
+			<font color={this.props.color} size={this.props.size}>{this.props.children}</font>
+		);
+	}
+}
+
+Title.propTypes = {
+	size: PropTypes.number.isRequired,
+	color: PropTypes.string.isRequired,
+};
```
Comment (wbobeirne) [inline js/components/Font.js]: Title would be undefined here, I think you want Font.

**Review by wbobeirne — CHANGES_REQUESTED** [review]

**Comment by xcatlkd** [comment]
Should work now.

**Review by wbobeirne — APPROVED** [review]
LGTM 👍

**Pull request closed without merging**